src/test/resources/features/SalesForceServiceSupport.feature

```diff
@@ -7,8 +7,19 @@ Feature: salesforce service support demo
     And login to salesforce
     Then click on service support button
     Then create new account
-    Then Edit Account Details
-    Then Delete Account
+#    Then Edit Account Details
+#    Then Delete Account
```

Comment:
Please review the commented lines and remove it if they are not required anymore